docs/kubernetes/windows.md

```diff
@@ -1,446 +1,358 @@
+- [Quick Start](#quick-start)
+    - [Install Needed Tools](#install-needed-tools)
+        - [Azure CLI](#azure-cli)
+        - [ACS-Engine](#acs-engine)
+        - [Kubectl for Windows](#kubectl-for-windows)
+    - [Generate SSH key](#generate-ssh-key)
+    - [Create a Resource Group and Service Principal](#create-a-resource-group-and-service-principal)
+    - [Create an acs-engine apimodel](#create-an-acs-engine-apimodel)
+    - [Generate Azure Resource Manager template](#generate-azure-resource-manager-template)
+    - [Deploy the cluster](#deploy-the-cluster)
+        - [Check that the cluster is up](#check-that-the-cluster-is-up)
+            - [SSH to the Linux master (optional)](#ssh-to-the-linux-master-optional)
+    - [Deploy your first application](#deploy-your-first-application)
+    - [What was deployed](#what-was-deployed)
+- [Next Steps](#next-steps)
 
-Note, with the rollout of new Windows Server versions in acs-engine, the workload deployed on Windows cluster should match as documented here: https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility . Both the containers being deployed,
-as well as the `kubletwin/pause` container must match the version of the Windows host. Otherwise, [pods may get stuck at ContainerCreating](https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#my-kubernetes-pods-are-stuck-at-containercreating) state.
+<!-- /TOC -->
 
-## Deployment
+## Quick Start
 
-Here are the steps to deploy a simple Kubernetes cluster with Windows:
+This guide will step through everything needed to build your first Kubernetes cluster and deploy a Windows web server on it. The steps include:
 
-1. [install acs-engine](../acsengine.md#downloading-and-building-acs-engine)
-2. [generate your ssh key](../ssh.md#ssh-key-generation)
-3. [generate your service principal](../serviceprincipal.md)
-4. edit the [Kubernetes windows example](../../examples/windows/kubernetes.json) and fill in the blank strings
-5. [generate the template](../acsengine.md#generating-a-template)
-6. [deploy the output azuredeploy.json and azuredeploy.parameters.json](../acsengine.md#deployment-usage)
+- Getting the right tools
+- Completing an ACS-Engine apimodel which describes what you want to deploy
+- Running ACS-Engine to generate Azure Resource Model templates
+- Deploying your first Kubernetes cluster with Windows Server nodes
+- Managing the cluster from your Windows machine
+- Deploying your first app on the cluster
 
-### Common customizations
+### Install Needed Tools
 
-As part of step 4, edit the [Kubernetes windows example](../../examples/windows/kubernetes.json), you can also make some changes to how Windows is deployed.
+#### Azure CLI
 
-#### Changing the OS disk size
+Click the [download](https://aka.ms/installazurecliwindows) link, and choose "Run". Click through the setup steps as needed.

```

Comment:
Add links for linux and macOS az  cli installs, too. Just in case they are a sysadmin tasked with managing both *nix and Windows applications and use an alternative platform for the Win K8s template generation / creation.